cert/cert.crt

```diff
@@ -0,0 +1,22 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
is pushing cert and private key the best thing we can do for security? doesn't seem like a good idea. what are our alternatives?